content/docs/command-reference/queue/start.md

```diff
@@ -12,39 +13,25 @@ usage: dvc queue start [-h] [-q | -v] [-j <number>]
 
 ## Description
 
-Starts one or more task queue worker processes. Each worker process will consume
-and execute one queued experiment task at a time in the background, until either
-`dvc queue stop` is used or the queue is empty.
+Starts one or more workers (`--jobs`) to process experiments. Each worker will
+consume and execute one queued experiment tasks at a time in the background,
+until either `dvc queue stop` is used or the queue is empty.
```

Comment:
- [ ] For a possible future follow up: review terms "task queue" and "worker process" everywhere. They're probably unnecessary IMO.